src/main/java/com/bruincreates/server/model/request/CreateReviewRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.bruincreates.server.model.request;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class CreateReviewRequest {
+
+    @NotNull
+    String username;
```

Comment:
We are supposed to get username inside the function instead for the frontend to pass in, otherwise, everyone can send fake requests from the frontend. You can check out examples in ProductController